backend/metasfresh-webui-api/src/main/java/de/metas/ui/web/dataentry/window/descriptor/factory/DataEntrySubTabBindingRepository.java

```diff
@@ -228,7 +228,7 @@ private void refreshFromDataEntryRecord(
 		final DocumentValuesSupplier documentValuesSupplier = new DataEntryDocumentValuesSupplier(
 				dataEntryRecord.get(),
 				dataEntryWebuiTools);
-		document.refreshFromSupplier(documentValuesSupplier);
+		document.refreshDataEntryFromSupplier(documentValuesSupplier);
```

Comment:
why simply calling the refresh is not ok?
what's the difference?

Reply:
* The refreshFromSupplier is calling ```de.metas.ui.web.window.model.DocumentChangesCollector.collectValueIfChanged(IDocumentFieldView, Object, ReasonSupplier)``` which only collects values if values were changed.
* I introduced the new option which instead calls ```de.metas.ui.web.window.model.DocumentChangesCollector.collectValueChanged(IDocumentFieldView, ReasonSupplier) ```. This one collects the values no matter what.

![image](https://user-images.githubusercontent.com/16001442/96770484-2a204e00-13e9-11eb-8f77-f750e281e60f.png)